report_aeroo/models/ir_actions_report_xml.py

```diff
@@ -61,21 +61,33 @@ def _get_default_outformat(self):
         help="Python expression used to determine the language "
         "of the record being printed in the report.",
         default="o.partner_id.lang")
+    company_eval = fields.Char(
+        'Company Evaluation',
+        help="Python expression used to determine the company "
+        "of the record being printed in the report.",
+        default="o.company_id")
 
     @api.multi
-    def get_template_from_lang(self, record, lang):
+    def get_template(self, record, lang, company):
         self.ensure_one()
 
         if not lang:
             lang = 'en_US'
 
-        line = next(
-            (l for l in self.report_line_ids if l.lang_id.code == lang), None)
+        if not company:
+            company = self.env.user.company_id
```

Comment:
@yasmineelmrini remove these 2 lines please. Always use the company evaluated from the field company_eval.